pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -253,10 +253,12 @@ func (ctrl *csiSnapshotSideCarController) checkandUpdateContentStatusOperation(c
 	var size int64
 	readyToUse := false
 	var driverName string
-	var snapshotID, groupSnapshotID string
+	var groupSnapshotID string
 	var snapshotterListCredentials map[string]string
 
-	if content.Spec.Source.SnapshotHandle != nil {
+	volumeGroupSnapshotMember := content.Status != nil && content.Status.VolumeGroupSnapshotHandle != nil
+
+	if content.Spec.Source.SnapshotHandle != nil || (volumeGroupSnapshotMember && content.Status.SnapshotHandle != nil) {
```

Comment:
How is (volumeGroupSnapshotMember && content.Status.SnapshotHandle != nil) sufficient to tell that this is statically provisioned?  Shouldn't we have content.Spec.Source.SnapshotHandle set already?

Reply:
In that case, you need to update the log msg on line 262 which talks about pre-bound.

Reply:
This is not changed?
```
klog.V(5).Infof("checkandUpdateContentStatusOperation: call GetSnapshotStatus for snapshot which is pre-bound to content [%s]", content.Name)
```

Reply:
Yeah, memory slip. I'm fixing it.